snippets/base/tests/test_views.py

```diff
@@ -107,6 +107,92 @@ def test_uuid_snippet(self):
 
 
 class FetchSnippetsTests(TestCase):
+    def test_base(self):
+        asrclient_kwargs = OrderedDict([
```

Comment:
You made a good point. This is copy / paste from other very old tests, I guess back in the day `urls.py` didn't use name parameters and thus we didn't it to be in the correct order to construct the `Client` object. 

I should remove those OrderedDicts anyway, since from Python 3.7 dicts are insertion ordered by default. Filed #1200 to keep in mind. Thanks!